client/src/components/Login/SignUpForm/index.js

```diff
@@ -85,7 +136,10 @@ const SignUpForm = ({ switchView }) => {
               value={email}
               placeholder="something@gmail.com"
               label="Email"
+              required
+              validateInput={validateUsername}
               changeHandler={handleChange}
+              validityState={validationState.username}
```

Comment:
It's not letting me add a comment where I want it, but you'll need to pass the validityState for the name field too.